theano/compile/profiling.py

```diff
@@ -1302,6 +1302,122 @@ def summary(self, file=sys.stderr, n_ops_to_print=20,
             print("-----------------", file=file)
             self.optimizer_profile[0].print_profile(file,
                                                     self.optimizer_profile[1])
+        self.print_tips()
+
+    def print_tips(self):
```

Comment:
Like for `summary_function`, `summary_globals`, and so on, we should be able to specify a destination file where the tips are printed. Can you add a `file` argument to `print_tips`, and then change all the `print(...)` calls to `print(..., file=file)`?